deployments/azure-functions-app-dotnet.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy DotNet project to Azure Function App
```

Comment:
It would be great for users to have some description about the template/workflow. What it does, how it does, what all is required for it to work. For example https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service.yml#L1-L28

Reply:
For all the templates in this PR

Reply:
There are some comments before the env block which tell how to set configuration. There is a help link as well. Is that sufficient?